test/index.test.ts

```diff
@@ -10,7 +10,7 @@ Object.defineProperty(window.document, "referrer", {
 import { actions, DispatchResponseEvent, createApp } from "../src";
 
 describe("createApp", () => {
-  const domain = "test-domain";
+  const domain = "http://test-domain.com/?id=appid";
```

Comment:
in dashboard we have `shop.domain.host` passed here, isn't that just a domain (without schema)? also as you pass ``src={urlJoin(src, `?domain=${shop.domain.host}&id=${appId}`)}`` in dashboard, should that be `test-domain.com&id=appid` here?